docs/experiments-msmarco-v2.md

```diff
@@ -27,6 +32,7 @@ sh target/appassembler/bin/IndexCollection -collection MsMarcoV2PassageCollectio
 
 Adjust `-threads` as appropriate.
 The above configuration, on a 2017 iMac Pro with SSD, takes around 30 minutes.
+On a 64-core Xeon 8160, eg `orca`, the full end-to-end run described below (all four datasets: indexing, searching dev/dev2, evaluation) takes around 1 hour 50 minutes with `-threads 32`.
```

Comment:
Let's not mention `orca` since it's a really old machine. Let's take about the "2017 iMac Pro" reference also?